back/engines/commercial/insights/app/models/insights/category_assignment.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Insights
+  class CategoryAssignment < ::ApplicationRecord
+    belongs_to :category
```

Comment:
You're not using a dependency to delete assignments when a category is deleted ? How is it done, then ?

Reply:
Very good point! I realised that this morning. It's also true for the idea-assignment relationship. To be fixed. I already fixed it for ideas in the other PR (didn't want to push new modifications here while u were reviewing it).